File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/authentication/JwtAuthenticationResponse.java
package com.javeriana.pontimovil.ponti_movil.dto.authentication;

import com.javeriana.pontimovil.ponti_movil.entities.Role;
import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class JwtAuthenticationResponse {

    // Atributos:
    private String token;
    private String email;
    private Role role;

    // Constructores:
    public JwtAuthenticationResponse() {
    }

    public JwtAuthenticationResponse(String token, String email, Role role) {
        this.token = token;
        this.email = email;
        this.role = role;
    }

}

File: frontend-angular/src/app/dto/authentication/JwtAuthenticationResponse.ts
import { Role } from '../../auth/models/role';

export class JwtAuthenticationResponse {
  token: string;
  email: string;
  role: string;

  constructor(
    token: string,
    email: string,
    role: string
  ) {
    this.token = token;
    this.email = email;
    this.role = role;
  }
}
